src/index.html

```diff
@@ -15,8 +15,91 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo-image"
+        href="index.html"
```

Comment:
The 'href' attribute of the <a> tag should not point to 'index.html' if it is meant to be a home link. Use './' or '/' for the home page to ensure the link always refers to the base of the website.